README.md

```diff
@@ -185,6 +185,7 @@ The following options can be set for any particular page. While the default opti
   * `name` - If the name of the thing that the page represents is different from the page title. (i.e.: "Frank's Café" vs "Welcome to Frank's Café")
   * `type` - The type of things that the page represents. This must be a [Schema.org type](http://schema.org/docs/schemas.html), and will probably usually be something like [`BlogPosting`](http://schema.org/BlogPosting), [`NewsArticle`](http://schema.org/NewsArticle), [`Person`](http://schema.org/Person), [`Organization`](http://schema.org/Organization), etc.
   * `links` - An array of other URLs that represent the same thing that this page represents. For instance, Jane's bio page might include links to Jane's GitHub and Twitter profiles.
+  * `date_modified` - An override for the `dateModified` field in the JSON-LD output. Useful when the file timestamp does not match the true time that the content was modified.
```

Comment:
(Comment from an earlier commit)
Google's Docs mention that the dateModified field is recommended. I can't give an exact explanation for its use other than it receives Google's recommendation.

Under this documentation, is listed as recommended for AMP pages, and ignored on non-AMP. But then again, it also says that datePublished is ignored on non-AMP pages. What are your thoughts?